### ISSUE TITLE
Can no longer hear music when playing to BM media_player on iPhone (app or browser)

### ISSUE THREAD

**Issue by Mariusthvdb** [thomasloven/hass-browser_mod#302]
My Home Assistant version: 2022.4.0.dev20220302

**What I am doing:**

Playing a local file to the browser_mod  media_player 

Odd thing is all was well, a couple of iterations of the HA beta/dev software ago, see: https://community.home-assistant.io/t/media-source-media-not-showing/229247/11 and https://community.home-assistant.io/t/migrating-media-files-from-config-local-to-media/395971

but, noticed 2 versions ago, local files are no longer being heard on the mobile devices, either the app, or a browser window on the phone, each with their own browser_mod id obviously.

The id and browser are recognized, because In the integrations panel I can see them turning form unavailable to idle/playing etc.

Also, I can play all audio files to my other media_players, so the syntax and paths to the files are 100% correct. I can even stream a radio station to the media_players, including the mobile devices! Even a tts message works ok.

Only thing not working is playing sound files to the mobile media_players.

The connection with the core HA media play software seems disturbed as of late. 
Another example of this is that when starting to play a radio stream, I can see the media_player popup and become 'playing'. However, if that stream is stopped, the player remains visible as 'playing'.

**What I expected to happen:**

sound file being heard

**What happened instead:**

Seems to be playing, but no sound

**Minimal** steps to reproduce:

```yaml
# The least amount of code or steps possible to reproduce my error
- service: media_player.play_media
  data:
    entity_id: media_player.phone_app
    media_content_id: >
      media-source://media_source/local/sounds/tunes/rome.mp3
    media_content_type: music
# Put your code/steps here
```

**Error messages from the browser console:**
no errors
```
// Select everything from the browser console and copy it

// Paste it here
```

this works (google home player):

<img width="944" alt="Schermafbeelding 2022-03-02 om 10 26 32" src="https://user-images.githubusercontent.com/33354141/156333639-266b4d12-ea13-47ac-9616-c86700f05d0d.png">

this doesnt work:

<img width="926" alt="Schermafbeelding 2022-03-02 om 10 27 06" src="https://user-images.githubusercontent.com/33354141/156333684-11da9034-509c-427b-86ee-a6d8af855e05.png">

---

**By replacing the space in the checkboxes ([ ]) with an X below, I indicate that I:**

- [X ] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [X ] Have made sure I am using the latest version of the plugin.

- [X ] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [ X] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by Mariusthvdb** [comment]
just found out that moving the files to share `/config` in /www folder again, and using my former scripts with path
```
media_content_id: /local/sounds/tunes/rome.mp3
```
again, immediately plays the files to the mobile devices...

So, I guess that means this isn't a browser-mod issues after all? And something got bugged in the HA software during the last few updates.

**Comment by Mariusthvdb** [comment]
fixed in 1.5.3!
thanks for that @gadgetchnnel
